app/views/items/_item.html.erb

```diff
@@ -1,4 +1,13 @@
 <div class=<%= "item-#{item.id}" %>>
-  <%= link_to item.name, user_item_path(item.user, item)%>
-  <%= link_to "",[item.user, item], method: :delete,remote: true, class: "glyphicon glyphicon-ok" %><br>
+  <div class="row">
```

Comment:
Great!  This is a much better approach then using `<br>` tags.  [Lists](http://getbootstrap.com/components/#list-group) are also an option, but don't work well for columns